cargo-near/src/commands/build_command/docker/metadata.rs

```diff
@@ -0,0 +1,129 @@
+use colored::Colorize;
+use serde::Deserialize;
+
+use crate::{types::metadata::CrateMetadata, util};
+use serde_json::Value;
+use std::collections::BTreeMap as Map;
+
+#[derive(Deserialize, Debug)]
+pub(super) struct ReproducibleBuild {
+    image: String,
+    image_digest: String,
+    pub container_build_command: Option<String>,
+    /// a string, containing https://git-scm.com/docs/git-clone#URLS,
+    /// currently, only ones, starting with `https://`, and ending in `.git` are supported
+    pub source_code_git_url: url::Url,
```

Comment:
Let's document in the README that cargo-near can use a custom section in Cargo.toml